doc/element_instance.tex

```diff
@@ -80,9 +80,26 @@
     </INSTANCE>
 </INSTANCE>
 \end{lstlisting}   
-   
+
+There one very specific case where a complex data type can be mapped as a single \texttt{ATTRIBUTE} (see section \ref{ATTRIBUTE}). 
+VODML comes with a data model (\texttt{ivoa}) providing basic datatypes. 2 of those (\texttt{ivoa:IntQuantity} 
+and \texttt{ivoa:RealQuantity}) that bind values with units. 
+As VODML encourages to use those types, MIVOT accept them to be serialized an single \texttt{ATTRIBUTE}-s
+instead of \texttt{INSTANCE}-s enclosing 2 individual attributes (one for the value and one for the unit).
+
```

Comment:
typos and suggested alternate phrasing.  I think having a complex REFERENCE including FOREIGN_KEY in this example is a bit distracting from the Quantity point.  Is there a different instance? or can we just snip the reference out (...)?

There is one very specific case where a complex datatype can be mapped as a single \texttt{ATTRIBUTE} (see section \ref{ATTRIBUTE}).
VODML includes a model (\texttt{ivoa}) of base datatypes.  The \texttt{ivoa:Quantity} types bind values with units and are designed to be directly compatible with the VOTable PARAM|FIELD elements.  MIVOT recommends that these types be mapped using a single \texttt{ATTRIBUTE} rather than as an \texttt{INSTANCE} containing individual attributes for value and unit.

Reply:
done